docs/apis-tools/frontend-development/01-introduction-to-task-applications.md

```diff
@@ -127,15 +123,16 @@ Often, task applications support the collaborative work on tasks, generally done
 
 Task applications are not limited to web applications to be worked on desktops. Camunda has been used successfully for the the development of omnichannel customer-facing applications, such as **mobile banking apps**, often via a **backend-for-frontend** implementation.
 
-<!--
-TODO Section to be added once pages are available
-
 ## Next steps
 
 You learned the basic concepts of a task application. Your possible next steps are:
 
-* Learn how to utilize Camunda 8 APIs to query and execute tasks in your task application, and to enrich it with process execution data.
-* Learn how to embed or customize Camunda Forms to render unique, tailored forms that can be designed by business users.
+- Learn how to [embed or customize Camunda Forms](/apis-tools/frontend-development/03-forms/01-introduction-to-forms.md) to render tailored forms that can be designed by business users.
+- Learn how to utilize [Camunda 8 APIs](/apis-tools/tasklist-api-rest/tasklist-api-rest-overview.md) to query and execute tasks in your task application, and to enrich it with process execution data.
+
+<!--
```

Comment:
Are we still waiting on this?

Reply:
@christinaausley anything else pending to be able to merge this?

Reply:
I'll have one more look over it and get this merged no later than tomorrow 👍